tests/spicy/tools/spicy-build-driver.spicy

```diff
@@ -1,7 +1,8 @@
 # @TEST-GROUP: no-jit
 # @TEST-EXEC: ${SPICY_BUILD} %INPUT
-# @TEST-EXEC: ${SCRIPTS}/printf '\x00\x00\x27\x10\x2a' | HILTI_DEBUG=spicy ./a.out -p Mini::Test >output
+# @TEST-EXEC: ${SCRIPTS}/printf '\x00\x00\x27\x10\x2a' | HILTI_DEBUG=spicy ./a.out -p Mini::Test -U >output
 # @TEST-EXEC: btest-diff output
+# @TEST-EXEC: grep -q "^# user_time" .stderr
```

Comment:
Might not be needed if we canonify time information instead of dropping it.

Reply:
I'll try the canonification but will still leave this in there as btest-diff is done only for debug builds.